N1990.Episerver.Cms.Audit/Controllers/VisitorGroupsController.cs

```diff
@@ -17,6 +18,7 @@ public VisitorGroupsController(ICmsAuditor cmsAuditor)
 	        _cmsAuditor = cmsAuditor;
 	    }
 
+        [Route("/EPiServer/cmsaudit/visitorgroups")]
```

Comment:
Not ideal with hardcoded urls - Better to support other roots than /episerver/ for edit-mode.

Reply:
Agree that module should support other roots that /episerver/. First commit had Paths.ToResource("CmsAudit", "CmsAudit")) but the actions on the controllers did not work.  An alternative is to change path to /cmsaudit/visitorgroups (remove episerver from the url) or getting the actions on the controller to work.